.storybook/addons.js

```diff
@@ -1,4 +1 @@
-// import '@kadira/storybook/addons'
-// import '@kadira/storybook-addon-knobs/register'
-// import '@kadira/storybook-addon-notes/register'
-// console.log('addons processed')
+import '@storybook/addon-knobs/register'
```

Comment:
Went ahead and enabled. Looks like use [needs to be added in stories themselves](https://github.com/storybooks/storybook/tree/master/addons/knobs#getting-started).